persistence/sql/persister_oauth2.go

```diff
@@ -177,12 +179,9 @@ func (p *Persister) ClientAssertionJWTValid(ctx context.Context, jti string) err
 func (p *Persister) SetClientAssertionJWT(ctx context.Context, jti string, exp time.Time) error {
 	return p.transaction(ctx, func(ctx context.Context, c *pop.Connection) error {
 		// delete expired
-		now := "now()"
-		if c.Dialect.Name() == "sqlite3" {
-			now = "CURRENT_TIMESTAMP"
-		}
+		nid := p.NetworkID(ctx)
 		/* #nosec G201 table is static */
-		if err := c.RawQuery(fmt.Sprintf("DELETE FROM %s WHERE expires_at < %s", oauth2.BlacklistedJTI{}.TableName(), now)).Exec(); err != nil {
+		if err := c.RawQuery(fmt.Sprintf("DELETE FROM %s WHERE expires_at < CURRENT_TIMESTAMP AND nid = ?", oauth2.BlacklistedJTI{}.TableName()), nid).Exec(); err != nil {
```

Comment:
Is `CURRENT_TIMESTAMP` supported everywhere? It looks like we used `now` before.

Reply:
`CURRENT_TIMESTAMP` is ANSI SQL and it is supported by all our db engines. `now` is a common alias for `CURRENT_TIMESTAMP`, but it isn't defined in sqlite3.